criticality_score/generate.py

```diff
@@ -28,10 +28,16 @@
     'go': ['Go'],
     'java': ['Java', 'Groovy'],
     'js': ['Javascript', 'Typescript', 'CoffeeScript'],
+    'julia': ['Julia'],
+    'matlab': ['Matlab'],
+    'nelson': ['Nelson'],
```

Comment:
i dont see Nelson, Octave, OpenMatrix in github languages.
Also, Matlab, scilab, julia, does it make sense to club in one category like scientific, computation or something ?